app/validators/answers/number_answer_validator.py

```diff
@@ -58,6 +65,15 @@ def validate_mandatory_has_no_default(self):
         if self.answer.get("mandatory") and self.answer.get("default") is not None:
             self.add_error(self.DEFAULT_ON_MANDATORY)
 
+    def validate_min_max_is_number(self):
+        min_value = self.answer.get("minimum", {}).get("value", 0)
+        max_value = self.answer.get("maximum", {}).get("value", 0)
```

Comment:
This isn't going to work for min/max that is referring to a value source, at the moment if a value source is 
``` 
value: {source: ..., identifier: ...}
```
the test passes because the type of value is dict, which is not a string, so this needs to be checking for a numeric type instead of not being a string, and may need to do something similar to runner for having a map of min & max